Cargo.toml

```diff
@@ -10,7 +15,7 @@ homepage = "https://talus.network"
 license = "Apache-2.0"
 readme = "README.md"
 authors = ["Talus Engineers <engineering@taluslabs.xyz>"]
-keywords = ["blockchain", "cli", "framework", "nexus", "sdk", "sui", "talus"]
+keywords = ["nexus", "sdk", "cli", "talus", "web3"]
```

Comment:
There is a warning here if there are > 5 keywords that we never addressed. I just did it  in this PR.